test/functional/sqlsrv/TC31_SimpleQuery.phpt

```diff
@@ -9,28 +9,28 @@ Basic verification of query statements (via "sqlsrv_query"):
 --ENV--
 PHPT_EXEC=true
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_versions_old.inc'); ?>
 --FILE--
 <?php
 
 require_once('MsCommon.inc');
 
-function SimpleQuery()
+function simpleQuery()
 {
     $testName = "Statement - Simple Query";
     startTest($testName);
 
     setup();
     $tableName = 'TC31test';
 
-    $conn1 = connect();
+    $conn1 = AE\connect();
 
-    createTable($conn1, $tableName);
+    // just create an empty table
+    $stmt1 = sqlsrv_query($conn1, "CREATE TABLE $tableName (dummyColumn int)");
 
     trace("Executing SELECT query on $tableName ...");
```

Comment:
I thought we were going to get rid of trace

Reply:
I don't mind keeping them around. Some tests have useful trace statements.

Reply:
But we never turn tracemode on.. If you think the output is useful, then use echo or print

Reply:
I personally do when I need to debug what goes wrong. If I use echo/print if will affect the expected output, which is often unnecessary.